DistroLauncher/WindowsUserInfo.cpp

```diff
@@ -41,48 +41,34 @@ namespace DistributionInfo {
         // for more YAML manipulation in the DistroLauncher arise, thus function should
         // be changed to use a proper YAML manipulation library, such as yaml-cpp.
         std::string WindowsUserInfo::toYamlUtf8() const {
-            // yaml-cpp doesn't support wide chars.
-            std::unordered_map<std::string,
-                std::unordered_map<std::string, std::string>> sections;
+            std::wstring fullYaml;
+
             if (!localeName.empty()) {
-                auto conv = Win32Utils::wide_string_to_utf8(localeName);
-                if (conv) {
-                    sections["Welcome"]["lang"] = conv.value();
-                } else {
-                    Helpers::PrintMessage(MSG_ERROR_CODE, HRESULT_FROM_WIN32(GetLastError()), conv.error().c_str());
+                fullYaml += L"Welcome:\n  lang: " + localeName + L'\n';
                 }
-            }
+
+            if (!(realName.empty() && userName.empty())) {
```

Comment:
it's a matter of taste but to me it would be easier to read as `!realName.empty || !username.empty()` :)

Reply:
I'll go with the recommendation.